Common/DTO/User.cs

```diff
@@ -1,8 +1,10 @@
-﻿namespace DTO
+﻿using System;
+
+namespace DTO
 {
     public class User
     {
-        public int Id { get; }
+        public Guid Id { get; }
         public string Login { get; }
         public string Email { get; set; }
         public string Password { get; }

```

Comment:
Still just Password not PasswordHash?

Reply:
I think this model would change after discussion. We have to add new properties. Do we have to change Password to PasswordHash in the UserCredential?